scan/lib/scan/options.rb

```diff
@@ -97,6 +97,12 @@ def self.available_options
                                      env_name: "SCAN_BUILDLOG_PATH",
                                      description: "The directory were to store the raw log",
                                      default_value: "#{FastlaneCore::Helper.buildlog_path}/scan"),
+        FastlaneCore::ConfigItem.new(key: :include_simulator_logs,
+                                     env_name: "SCAN_INCLUDE_SIMULATOR_LOGS",
+                                     description: "Should the logs of the app on the Simulator be written to the output_directory",
```

Comment:
[This](https://github.com/soffes/SAMKeychain/issues/52) is such an issue that I solved by modifying a copy of `scan` on our Jenkins CI slave to produce the logs.